proxy/core/src/main/scala/io/cloudstate/proxy/AkkaActorRegisterFeature.scala

```diff
@@ -8,26 +8,23 @@ import scala.collection.JavaConverters._
 
 @AutomaticFeature
 final class AkkaActorRegisterFeature extends Feature {
-
-  override def duringAnalysis(access: Feature.DuringAnalysisAccess) =
-    reachableSubtypes(access, access.findClassByName("akka.akka.Actor")).foreach { cls =>
```

Comment:
@dwijnand A note about the thing I solved with the class constants—see the typo?

Reply:
I spy, with my little eye...